test/e2e/ethtransfer_test.go

```diff
@@ -31,15 +31,15 @@ func Test1000EthTransfer(t *testing.T) {
 	require.NoError(t, err)
 
 	// Load account with balance on local genesis
-	auth, err := operations.GetAuth("0x59c6995e998f97a5a0044966f0945389dc9e86dae88c7a8412f4603b6b78690d", big.NewInt(1001))
+	auth, err := operations.GetAuth("0x59c6995e998f97a5a0044966f0945389dc9e86dae88c7a8412f4603b6b78690d", big.NewInt(1000))
 	require.NoError(t, err)
 	// Load eth client
 	client, err := ethclient.Dial("http://localhost:8123")
 	require.NoError(t, err)
 	// Send txs
-	nTxs := 100
+	nTxs := 1000
```

Comment:
Now that the test is running on GHA and it waits for batches to be consolidated, the `1000` should be reduced. Otherwise the test timeout (timeout currently set to 10min). 

It's possible to increase the timeout, but IMO it's better to make the test faster, I'd try to change  `nTxs` to 100 or 200.

Also if you introduce this change, it would make sense to change the name of the test to `Test1000EthTransfer => TestEthTransfer`